test/cmd/deployments.sh

```diff
@@ -16,8 +16,14 @@ oc create -f test/integration/fixtures/test-deployment-config.json
 oc describe deploymentConfigs test-deployment-config
 [ "$(oc get dc -o name | grep 'deploymentconfig/test-deployment-config')" ]
 [ "$(oc describe dc test-deployment-config | grep 'deploymentconfig=test-deployment-config')" ]
+# Patch a nil list
+oc env dc/test-deployment-config TEST=value
```

Comment:
can we add checks to make sure the env command 1) actually took effect and 2) didn't change anything else?